### PR TITLE
[5.6] Make request accessible in password reset responses

### PR DESCRIPTION
**Pull request by hafezd**
It's necessary to access `$request` for customization of `sendResetResponse()` in `\Illuminate\Foundation\Auth\ResetsPasswords` and `sendResetLinkResponse()` in `\Illuminate\Foundation\Auth\SendsPasswordResetEmails`.

### CODE REVIEW

**Comment by m1guelpf** [comment]
This is breaking and should go to master

File: src/Illuminate/Foundation/Auth/SendsPasswordResetEmails.php [hunk 1]
```diff
@@ -53,10 +53,11 @@ protected function validateEmail(Request $request)
     /**
      * Get the response for a successful password reset link.
      *
+     * @param  \Illuminate\Http\Request
```
Comment (GrahamCampbell) [inline src/Illuminate/Foundation/Auth/SendsPasswordResetEmails.php]: Missing param name. :)

**Review by GrahamCampbell — CHANGES_REQUESTED** [review]

File: src/Illuminate/Foundation/Auth/SendsPasswordResetEmails.php (hunk 1 above)
Comment (hafezdivandari) [reply src/Illuminate/Foundation/Auth/SendsPasswordResetEmails.php]: Missing param names added.

**Comment by gabriel-caruso** [comment]
@hafezdivandari Rebase your branch, it would avoid many commits to `master` :smile:

**Comment by taylorotwell** [comment]
Honestly you can just use the `request()` helper rather than us making this breaking change.

**Pull request closed without merging**